module/VuFind/src/VuFind/View/Helper/Root/AlphaBrowse.php

```diff
@@ -92,6 +94,9 @@ public function getUrl($source, $item)
         if ($item['count'] == 1) {
             $query['jumpto'] = 1;
         }
+        if (isset($origin)) {
```

Comment:
You could make this a one-liner as:

```
$query += $origin?->getSearchUrlParamsArray() ?? []
```

...if you want to. It's fine as it is if you prefer it this way, though. (An intermediate simplification would be to change `if (isset($origin)) {` to simply `if ($origin) {` since a non-null value will always evaluate to truthy in this situation).